tests/unit/nupic/algorithms/knn_classifier_test.py

```diff
@@ -29,6 +29,30 @@
 
 class KNNClassifierTest(unittest.TestCase):
 
+  def testSparsifyVector(self):
+    classifier = KNNClassifier(distanceMethod="norm", distanceNorm=2.0)
```

Comment:
Move this line down to under line 36 for consistency with the rest of the code blocks.